system-tests/src/tests/organization-list.spec.ts

```diff
@@ -1,11 +1,80 @@
-import { test } from "@playwright/test"
+import { expect, test } from "@playwright/test"
 
 import expectScreenshotsToMatchSnapshots from "../utils/screenshot"
 
+import { manageOrganization } from "@/utils/organizationUtils"
+
 test.use({
   storageState: "src/states/admin@example.com.json",
 })
 
+test("create new organization, edit it and it's permissions, and delete it", async ({ page }) => {
+  await page.goto("http://project-331.local/")
+  await page.getByRole("link", { name: "All organizations" }).click()
+
+  // Create organization
```

Comment:
You could convert these comments that group stages of the tests into `test.step`'s https://playwright.dev/docs/api/class-test#test-step